src/controller/python/chip/crypto/p256keypair.cpp

```diff
@@ -0,0 +1,85 @@
+
+#include <controller/python/chip/crypto/p256keypair.h>
+
+#include <cassert>
+#include <lib/support/logging/CHIPLogging.h>
+
+using namespace chip;
+using namespace chip::python;
+using namespace chip::Crypto;
+
+pychip_P256Keypair::pychip_P256Keypair(void * aPyContext, pychip_P256Keypair_ECDSA_sign_msg aSignMsgFunct,
+                                       pychip_P256Keypair_ECDH_derive_secret aDeriveSecretFunct) :
+    mPyContext(aPyContext),
+    mSignMsgFunct(aSignMsgFunct), mDeriveSecretFunct(aDeriveSecretFunct)
+{}
+
+pychip_P256Keypair::~pychip_P256Keypair() {}
+
+CHIP_ERROR pychip_P256Keypair::Initialize(Crypto::ECPKeyTarget key_target)
```

Comment:
if this is empty, why do we have it?

Reply:
To override the default Initialize() function from the platform.